libdvdcss.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "libdvdcss",
```

Comment:
this should be in the main manifest

Reply:
no I mean copy the module from here to the main manifest and delete this extra manifest

Reply:
@daviewales you have to remove this now, it's a leftover.

Reply:
Could you clarify? Do you mean I should remove the `"name": "libdvdcss", ` line? Or something else?

Reply:
removing this file.

Reply:
Thanks. Done.